ONSdigital/rasrm-acceptance-tests
# Social sample file pilot changes

# Motivation and Context

Due to the format of the social sample file changing we need to update the acceptance tests to assert updated columns and the data presented back

# What has changed



Added new social sample file and updated social acceptance tests with new asserts for data

# How to test?



Run `make acceptance_tests` and ensure they pass against all the `social-sample-file-pilot-changes` branches for the associated repo's (These are all linked on the trello card - https://trello.com/c/3PK5xIDM)

# Links




- https://github.com/ONSdigital/response-operations-ui/pull/220
- https://github.com/ONSdigital/respondent-home-ui/pull/41
- https://github.com/ONSdigital/rm-sample-service/pull/72
- https://github.com/ONSdigital/rm-actionsvc-api/pull/28
- https://github.com/ONSdigital/rm-action-service/pull/74
- https://github.com/ONSdigital/rm-samplesvc-api/pull/17
